core/src/main/resources/lib/layout/ajax.jelly

```diff
@@ -42,6 +42,7 @@ THE SOFTWARE.
       evaluating a page that has called l:layout tag.
     </st:attribute>
   </st:documentation>
+  <st:setHeader name="Cache-Control" value="no-cache,no-store,must-revalidate" />
```

Comment:
BTW space before `/>` is not needed in XML, only in `XHTML` that might be parsed by a noncompliant browser.